taprpc/marshal.go

```diff
@@ -444,7 +444,8 @@ func UnmarshalGroupWitness(wit *GroupWitness) (*asset.PendingGroupWitness,
 
 // MarshalAsset converts an asset to its rpc representation.
 func MarshalAsset(ctx context.Context, a *asset.Asset,
-	isSpent, withWitness bool, keyRing KeyLookup) (*Asset, error) {
+	isSpent, withWitness bool, keyRing KeyLookup,
+	decDisplay uint32) (*Asset, error) {
```

Comment:
See comment. Above, I think we should either make this an option, and/or modify the proto to be its own message, that way we can check for nil vs not.